### PR TITLE
chore(primitives): use alloy eth conversion constants

### PR DESCRIPTION
**Pull request by tcoratger**

### CODE REVIEW

**Review by mattsse — APPROVED** [review]

**Review by onbjerg — APPROVED** [review]
lgtm, needs a rebase though (sorry about that)

**Comment by tcoratger** [comment]
> lgtm, needs a rebase though (sorry about that)

@onbjerg no problem, done :)

File: crates/ethereum/evm/src/strategy.rs [hunk 1]
```diff
@@ -0,0 +1,1174 @@
+//! Ethereum block execution strategy,
```
Comment (onbjerg) [inline crates/ethereum/evm/src/strategy.rs:1]: needs to be removed, it's not on main

**Review by onbjerg — DISMISSED** [review]

File: crates/ethereum/evm/src/strategy.rs (hunk 1 above)
Comment (tcoratger) [reply crates/ethereum/evm/src/strategy.rs]: oups done

**Comment by tcoratger** [comment]
@onbjerg Should be ready now, sorry for back and forth

**Review by emhane — APPROVED** [review]

**Review by mattsse — APPROVED** [review]

**Pull request merged**